network/p2p/tracer/gossipSubMeshTracer.go

```diff
@@ -450,6 +479,22 @@ func (t *GossipSubMeshTracer) LastHighestIHaveRPCSize() int64 {
 	return t.rpcSentTracker.LastHighestIHaveRPCSize()
 }
 
+// DuplicateMessageCount returns the current duplicate message count for the peer.
+func (t *GossipSubMeshTracer) DuplicateMessageCount(peerID peer.ID) float64 {
+	count, found, err := t.duplicateMessageTrackerCache.GetWithInit(peerID)
+	if err != nil {
+		t.logger.Err(err).
+			Bool(logging.KeyNetworkingSecurity, true).
+			Str("peer_id", peerID.String()).
+			Msg("failed to get duplicate message count for peer")
+		return 0
```

Comment:
is it safe to return 0 on error?

Reply:
https://github.com/onflow/flow-go/pull/5058/commits/4cea2dfabc3719d05c548fc81e0018e4f442b84b